src/config.js

```diff
@@ -34,52 +34,50 @@ module.exports = (store) => {
      * @param {AbortSignal} [options.signal] - abort this config read
      * @returns {Promise<Config>}
      */
-    getAll (options = {}) { // eslint-disable-line require-await
-      return configStore.get(undefined, options)
+    async getAll (options = {}) { // eslint-disable-line require-await
+      // level-js@5.x cannot read keys from level-js@4.x dbs so fall back to
```

Comment:
Is there an issue to track this? If not, can you create one?

Reply:
I can do, but it's not something that's ever going to be fixed so the issue will hang around forever.

The 4 -> 5 version bump was because of this breaking change.

Reply:
ok makes sense, so maybe not worth